tests/contrib/operators/test_ecs_operator.py

```diff
@@ -63,26 +65,26 @@ def setUp(self, aws_hook_mock):
         configuration.load_test_config()
 
         self.aws_hook_mock = aws_hook_mock
-        self.ecs = ECSOperator(
-            task_id='task',
-            task_definition='t',
-            cluster='c',
-            overrides={},
-            aws_conn_id=None,
-            region_name='eu-west-1',
-            group='group',
-            placement_constraints=[
-                {
-                    'expression': 'attribute:ecs.instance-type =~ t2.*',
-                    'type': 'memberOf'
-                }
-            ],
-            network_configuration={
+        self.ecs_operator_args = {
+            'task_id': 'task',
+            'task_definition': 't',
+            'cluster': 'c',
+            'overrides': {},
+            'aws_conn_id': None,
+            'region_name': 'eu-west-1',
+            'group': 'group',
+            'placement_constraints': [{
+                'expression': 'attribute:ecs.instance-type =~ t2.*',
+                'type': 'memberOf'
+            }],
+            'network_configuration': {
                 'awsvpcConfiguration': {
                     'securityGroups': ['sg-123abc']
                 }
             }
-        )
+        }
+        self.ecs = ECSOperator(**self.ecs_operator_args)
+        self.ecs_fargate = ECSOperator(launch_type='FARGATE', **self.ecs_operator_args)
```

Comment:
Do we need to instantiate a new fargate operator before each test if only one test uses it? What do you think about setting `self.launch_type` in the relevant tests instead?

Reply:
If we want to create `ecs_fargate` in the specify test then we need mock AwsHook again and use the new mock instead `self.aws_hook_mock`, but of course it not a problem